setup.py

```diff
@@ -0,0 +1,3 @@
+import setuptools
+
+setuptools.setup()
```

Comment:
You may want to look into using PBR.

https://github.com/sc68cal/Kostyor/blob/master/setup.cfg

https://github.com/sc68cal/Kostyor/blob/master/setup.py